### ISSUE TITLE
FRBRoo namespaces: multiple versions of CIDOC-CRM

### ISSUE THREAD

**Issue by pasqLisena** [DOREMUS-ANR/doremus-ontology#3]
Namespaces in the [_owl_ file](https://github.com/DOREMUS-ANR/doremus-ontology/blob/master/FRBRoo-v2.4-Biblissima-2016-04-17.owl):

- CIDOC-CRM **`http://erlangen-crm.org/current/`**
    - e.g. `http://erlangen-crm.org/current/E73_Information_Object`
- FRBR **`http://erlangen-crm.org/efrbroo/`**
    - e.g. `http://erlangen-crm.org/efrbroo/F22_Self-Contained_Expression`

Namespaces we currently use in marc2rdf:
- CIDOC-CRM **`http://www.cidoc-crm.org/cidoc-crm/`**
- FRBR **`http://erlangen-crm.org/efrbroo/`**

Why this difference in CIDOC-CRM?

**Comment by rtroncy** [comment]
Tentative explanation (according to LOV):
* we have a version of CIDOC-CRM (latest is v5.0.4) defined at http://www.cidoc-crm.org/cidoc-crm/ with the prefix ```crm``` (see also http://lov.okfn.org/dataset/lov/vocabs/crm)
* we have the erlanger version of CIDOC-CRM (latest is 5.1.2) defined at http://erlangen-crm.org/current/ with the prefix ```ecrm``` (see also http://lov.okfn.org/dataset/lov/vocabs/ecrm)
According to prefix.cc, ```crm``` is also associated to multiple namespaces, see http://prefix.cc/crm
This is clearly a touchy issue. The fact that there is an official and several non-official versions of CIDOC-CRM. We need to take a decision based on maturity and usage.
@pierrechoffe Do you have more inside information about the various versions of CIDOC-CRM that exist?

**Comment by pierrechoffe** [comment]
@rtroncy 
>@pierrechoffe Do you have more inside information about the various versions of CIDOC-CRM that exist?

No I don't but I will ask
FYI I recently asked about Erlangen's work on FRBRoo v2.4 but got no answer. Will ask again.

**Comment by pierrechoffe** [comment]
@pasqLisena @rtroncy update:
I never received any answer to my question. I will attend the next CRM SIG meeting in the first week of August and will ask again. If you have complementary questions, I don't hesitate.

**Comment by pasqLisena** [comment]
Thanks Pierre, so let's try to have in August a clear idea about which version (URI) we are going to use.

**Comment by pasqLisena** [comment]
Hi @pierrechoffe, have you news on this point?
Thank you 😄

**Comment by pierrechoffe** [comment]
Hi @pasqLisena the crm team is not clear about versioning. The answer is the version of CIDOC-CRM (latest is v5.0.4) defined at http://www.cidoc-crm.org/cidoc-crm/ with the prefix crm is the rdf version of the documentation. This is a fixed version, already ancient.

The erlangen version of CIDOC-CRM (latest is v5.1.2) defined at http://erlangen-crm.org/current/ with the prefix ecrm is the OWL version. It is supposed to be updated regularly.

I assume we should use the erlangen version.

**Comment by pasqLisena** [comment]
Good. I agree also because we still have the enlargen version in FRBRoo.owl.

I close the issue.

**Comment by rtroncy** [comment]
Correction @pierrechoffe OWL is RDF too. What you meant is:
* The CIDOC-CRM ontology is encoded in **RDFS**.  There is this [version 6.2.1](http://www.cidoc-crm.org/rdfs/cidoc_crm_v6.2.1-draft-b-2015October.rdfs) from October 2015 as well. The [version 5.0.4](http://lov.okfn.org/dataset/lov/vocabs/crm) is the latest archived by LOV
* The erlangen ontology is represented in **OWL DL**. Why are you saying the latest is v5.1.2 since the metadata at http://erlangen-crm.org/current/ states that it is based on v6.2.1 ?

It would be good if someone (@pasqLisena?) makes an assessment of the extra axiomatization introduced in the OWL version. At the minimum, produce a document counting the number of classes / properties defined in each version (crm and ecrm), and counting the number of additional restrictions or axioms introduced.

**Comment by pierrechoffe** [comment]
>Why are you saying the latest is v5.1.2 since the metadata at http://erlangen-crm.org/current/ states that it is based on v6.2.1 ?

@rtroncy sorry this was a copy and paste of your message of 15 June, I did not check whether there was an update. If we have 6.2.1 this is the ultimate one so far.

**Comment by rtroncy** [comment]
I noticed this afterwards, and this proves that erlangen is regularly updating this http://erlangen-crm.org/current/ url, good sign :-)

**Comment by pasqLisena** [comment]
### ENLARGEN:
84 classes,   242 properties
**exclusive classes** :    nothing
**exclusive properties** :   P69i_is_associated_with, P69_has_association_with


### CIDOC:
84 classes,   286 properties
**exclusive classes** :    nothing
**exclusive  properties** :    P3_has_note, P10_falls_within, P10i_contains, P22_transferred_title_to, P22i_acquired_title_through, P27_moved_from, P27i_was_origin_of, P57_has_number_o
f_parts, P69_is_associated_with, P79_beginning_is_qualified_by, P80_end_is_qualified_by, P81_ongoing_throughout, P82_at_some_time_within, P86_falls_within, P86i_contains, P89_falls
_within, P89i_contains, P90_has_value, P106_is_composed_of, P106i_forms_part_of, P114_is_equal_in_time_to, P115_finishes, P115i_is_finished_by, P116_starts, P116i_is_started_by, P1
17_occurs_during, P117i_includes, P120_occurs_before, P120i_occurs_after, P121_overlaps_with, P122_borders_with, P127_has_broader_term, P127i_has_narrower_term, P132_overlaps_with,
 P133_is_separated_from, P148_has_component, P148i_is_component_of, P150_defines_typical_parts_of, P150i_defines_typical_wholes_for, P156i_is_occupied_by, P168_place_is_defined_by,
 P168i_defines_place, P81a_end_of_the_begin, P81b_begin_of_the_end, P82a_begin_of_the_begin, P82b_end_of_the_end


In summary, CIDOC uses a unique **P69** in both direction, and adds lots of properties.

**Comment by rtroncy** [comment]
Thanks @pasqLisena, excellent work!

By exclusive properties, you meant a property that exists with this label in one ontology but not in the other, right? This means that both CIDOC and ERLANGEN have exactly the same list of 84 classes. I assume the hierarchy is also the  same, except that ERLANGEN being in OWL, it might add additional ```owl:Restriction``` axioms to define the classes.

We probably don't need to use **P69** in both directions. On the other side, the 46 new properties introduced in CIDOC might of interest for us, no? So, are we sure we don't want to be based on the CIDOC (RDFS) ontology rather than on the ERLANGEN (OWL) ontology? Is it because FRBRoo is based on ERLANGEN? If so, why did they make this choice?

@pierrechoffe this can be questions you may want to ask to the CRM SIG meeting  ... is it this week?

**Comment by pasqLisena** [comment]
> By exclusive properties, you meant a property that exists with this label in one ontology but not in the other, right?

Yes, I confirm.
-----
Sorry but I did a mistake in the extraction of this information (I was considering only ObjectProperties in enlargen). This is the real list

### ENLARGEN:
84 classes,   280 properties
**exclusive classes** :    nothing
**exclusive properties** :   P69i_is_associated_with, P69_has_association_with


### CIDOC:
84 classes,   286 properties
**exclusive classes** :    nothing
**exclusive  properties** :    P22i_acquired_title_through, P69_is_associated_with, P156i_is_occupied_by, P168i_defines_place, P81a_end_of_the_begin, P81b_begin_of_the_end, P82a_begin_of_the_begin, P82b_end_of_the_end


I realized because was weird the absence of P3 and P10 because they are of common use.

**Comment by rtroncy** [comment]
Ah, this changes the situation indeed. We have the notion of P69 in both ontologies. CIDOC is introducing some new inverse properties that we do not really care. The only novel properties are those P81a, P81b, P82a and P82b (I love their names) ... do we need them? I assume not ... and I better understand why we can decide to base DOREMUS on ERLANGEN.

P.S: I have edited your messages @pasqLisena since you're confusing how to write **erlangen** (not enlargen) :-)

**Comment by pierrechoffe** [comment]
@pasqLisena @rtroncy we do not need these P81a etc properties, CRM is presently working on much more precise ways of describing intervals of time and relations between spacio-temporal entities

**Comment by pierrechoffe** [comment]
@pasqLisena @rtroncy P69_has_association is a symmetric property, we use a subproperty of it between M6_Casting when we have alternate castings (they are "alternate" of each other)

**Comment by pasqLisena** [comment]
So, can we say that we are going to use **erlangen** version? (If yes, I'll update marc2rdf)

**Comment by pierrechoffe** [comment]
our version of FRBRoo 2.4 will be realised by erlangen and they will most probably refer to the erlangen crm/current version, so I assume this is the right choice

**Comment by rtroncy** [comment]
**Decision:** We will base the DOREMUS ontology on http://erlangen-crm.org/current/ (prefix: ecrm)

**Comment by pasqLisena** [comment]
Updated [marc2rdf](https://github.com/DOREMUS-ANR/marc2rdf/commit/ee078104b3ab2612c4a6704ea165844495174c87).
